getting-started/kubernetes/vpp/getting-started.md

```diff
@@ -0,0 +1,199 @@
+systemctl restart kubelet
+````
+
+#### Configure nodes for VPP
+
+Start by getting the appropriate yaml manifest for the {{ site.prodname }} VPP dataplane:
+```bash
+# If you have configured hugepages on your machines
+curl -o calico-vpp.yaml https://raw.githubusercontent.com/projectcalico/vpp-dataplane/master/yaml/generated/calico-vpp.yaml
+```
+```bash
+# If not, or if you're unsure
+curl -o calico-vpp.yaml https://raw.githubusercontent.com/projectcalico/vpp-dataplane/master/yaml/generated/calico-vpp-nohuge.yaml
+```
+
+Then configure these parameters in the `calico-config` ConfigMap in the yaml manifest.
+
+**Required**
+
+* `vpp_dataplane_interface` is the primary interface that VPP will use. It must be the name of a Linux interface, configured with an address. The address configured on this interface must be the node address in Kubernetes (`kubectl get nodes -o wide`).
+* service_prefix is the Kubernetes service CIDR. You can retrieve it by running:
+````bash
+kubectl cluster-info dump | grep -m 1 service-cluster-ip-range
+````
+If this command doesn't return anything, you can leave the default value of `10.96.0.0/12`.
+
+**Optional**
+
+* `default_ipv4_pool_cidr` allows you to customize the default {{ site.prodname }} IPv4 pool. For details, see [reference]({{ site.baseurl }}/reference/node/configuration)
+* `vpp_uplink_driver` configures how VPP grabs the physical interface, available values are :
+  * `""` : will automatically select and try drivers based on available resources, starting with the fastest
+  * `avf` : use the native AVF driver.
+  * `virtio` : use the native virtio driver (requires hugepages).
+  * `af_xdp` : use an AF_XDP socket to drive the interface (requires kernel 5.4 or newer).
+  * `af_packet` : use an AF_PACKET socket to drive the interface (slow but works everywhere).
+  * `none` : do not configure connectivity automatically. This can be used when [configuring the interface manually]({{ site.baseurl }}/reference/vpp/uplink-configuration).
+
+**Example**
+
+````yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: calico-config
+  namespace: kube-system
+data:
+  service_prefix: 10.96.0.0/12
+  vpp_dataplane_interface: eth1
+  default_ipv4_pool_cidr: 172.16.0.0/16
+  vpp_uplink_driver: ""
+  ...
+````
+
+#### Apply the configuration
+
+To apply the configuration, run:
+````bash
+kubectl apply -f calico-vpp.yaml
+````
+
+This will create all the resources necessary to connect your pods through VPP and configure the CNI on the nodes.
+
+### Next steps
+
+After installing {{ site.prodname }} with the VPP dataplane, you can benefit from the goodies of the VPP dataplane, such as fast [IPsec]({{ site.baseurl }}/getting-started/kubernetes/vpp/ipsec) or [Wireguard]({{ site.baseurl }}/security/encrypt-cluster-pod-traffic) encryption.
+
+**Tools**
+
+- [Install and configure calicoctl]({{site.baseurl}}/getting-started/clis/calicoctl/install) to configure and monitor your cluster.
```

Comment:
Not sure this makes sense any longer given we support kubectl...Casey?

Reply:
We can leave it for now - at some point I will be doing a comprehensive sweep of the docs to adjust this.

For now, kubectl is in tech-preview anyway and there are a few more operations that need to be migrated.